hyperspy/_components/gaussian2d.py

```diff
@@ -45,7 +45,7 @@ class Gaussian2D(Expression):
     Parameters
     ----------
     A : float
-        Amplitude (height of the peak scaled by :math:`2 \pi s_x s_y`).
+        Area (height of the peak scaled by :math:`2 \pi s_x s_y`).
```

Comment:
I am wondering if `area` is a good wording for a 2D gaussian?

Reply:
True, `volume` would be correct. Changed and added a statement that it is equivalent to the area in a 1D Gaussian.